src/components/Header/Notifications/Modal/Modal.js

```diff
@@ -0,0 +1,139 @@
+    return (
+      <div className="container">
+        <div className="card">
+          <div id="modal" className={showModal ? '' : 'hide'}>
+            <form>
+              <div className="modal-content">
+                <span className="close" onClick={closeModal}>
+                  &times;
+                </span>
+                <div className="text-black center-align">
+                  <h4>Notification</h4>
+                </div>
+                <br />
+                <div />
+                {(notifications || []).map((val, key) => (
+                  <div key={key} className={val.status === 'unseen' ? 'row light-red' : 'row'}>
+                    <div key={key}>
+                      <div role="button" onClick={() => this.updateStatus(val)}>
+                        <div key={key} className="wrap-notification">
+                          <div className="notification-tick">
+                            {val.status === 'unseen' ? (
+                              <FontAwesomeIcon
+                                icon={faCheck}
+                                size="1x"
+                                className="text-light-grie small-padding left"
+                              />
+                            ) : (
+                              <FontAwesomeIcon
+                                icon={faCheckDouble}
+                                size="1x"
+                                className="left small-padding text-light-grey"
+                              />
+                            )}
+                            <FontAwesomeIcon
+                              icon={faTrash}
+                              size="1x"
+                              id={`deleteNotification${key}`}
+                              className="left small-padding text-danger"
+                              onClick={() => this.deleteNotificat(val)}
+                            />
+                          </div>
+                          <div className="left medium-text notification-message ">
+                            <a className="text-info small-v-margin" href={val.url}>
+                              Click here to read
+                            </a>
+                            <br />
+
+                            {val.message.length > 120
+                              ? `${val.message.substring(0, 120)}...`
+                              : `${val.message}`}
+                            <br />
+                          </div>
+
+                          <div className="right small-text text-light-grie notification-time hide-on-medium hide-on-small">
+                            {new Date(val.createdAt).toDateString()}
+                          </div>
+                        </div>
+                      </div>
+                      <div className="divider" />
+                    </div>
+
+                    <div className="divider light-grey" />
+                  </div>
+                ))}
```

Comment:
Put a message like `No new notification`, `No notification right now` or, `No notifications yet` when there is no notification

Reply:
Done